src/test/java/com/offdk/play/web/UserParserTest.java

```diff
@@ -0,0 +1,26 @@
+package com.offdk.play.web;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+
+import com.offdk.play.model.slack.User;
+
+public class UserParserTest {
```

Comment:
Add additional test cases to follow all possible logic. For example, if you are parsing a list, it might make sense to have three test cases: 1) Empty list, 2) List with one element, 3) List with multiple elements